2-transform-data/bq_ingestion/extract_msgs_test.py

```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
             got_json = em.convert_msg_to_json(test["msg"])
             self.assertEqual(want_json[key], got_json, "Convert message to json error")
 
+    
+    @mock.patch("google.cloud.storage.Client")
```

Comment:
Change to google.cloud.bigquery.Client and mock.patch any other objects that are used in the function here like Table.